### ISSUE TITLE
Updates should be handled only once

### ISSUE THREAD

**Issue by GiuseppeFn** [HeySreelal/televerse#84]
At the moment, in a situation like this:
```
bot.command('help', help);
bot.onMessage.listen(onMessage);
```
The update will be handled both by onMessage and help, and this probably shouldn't be the case (But as the updates are listened from an "update stream" this behavior maybe is wanted), but could at least be deactivated

IF THE BEHAVIOR ISN'T WANTED, you should refactor the code, by removing the updates stream, and instead, in the "start" function, you should listen to the long polling, and if you receive the updates, you should iter in each handler, and as soon one handler's filters match, you should run it and then break the for loop

PLUS, i suggest, if you do those modifications, you should make the "start" function blocking by using an await for while waiting for updates, as a long polling function should be blocking.

**Comment by HeySreelal** [comment]
Hello again! I'm aware of this issue, but I think it's okay to stream every message into the `onMessage` stream than pause it because the `command` or any other method is streaming the same update.

Maybe I need to brainstorm a little more.

Thanks again for sharing the idea.

**Comment by HeySreelal** [comment]
Hey, just wanted to update this behavior is implemented to `Televerse v1.9.0`. 

Televerse now only relies on the `onUpdate` method on the Fetcher method to listen to updates. Televerse parallelly keeps track of all the handlers set up in an internal list.

As part of this change, I had to completely remove the `Event` class from the library. Since we're only relying on Fetcher, there's no need to create an extra complexity with the Event class. All the getters in the Event class have now been moved to Televerse class as methods.

Thanks again for suggesting this :)
